### PR TITLE
fix: Remove autologin in Keycloak in E2EI [WPB-7061]

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-7061" title="WPB-7061" target="_blank">BugWPB-7061</a>  [Android] Crash when user tries to register certificate for another email
  </td></table>
  <br />
 

# What's new in this PR?

### Issues

Keyclok and WebView that opens it cache users creds. So if user logs in with the wrong creds in Keyclok and tries to enroll E2EI certificate, he gets an error and no chance to change it and get a certificate (until cache is cleared).

### Solutions

add `AuthorizationRequest.Prompt.LOGIN` parameter into `AuthorizationRequest`, it makes Keycloak force user to enter password (even when the creds are cached) **OR** sing out and sign in with another creds.

### Attachments (Optional)

Screenshot 2024-03-08 at 16 20 31

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [3507](https://10.10.124.134/job/AR-build-pipeline/3507/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.2(9094188)(9094188)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.2(9094188)(9094188)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/e2eiEnrollment/GetE2EICertificateViewModel.kt [hunk 1]
```diff
@@ -68,8 +68,7 @@ class GetE2EICertificateViewModel @Inject constructor(
                     .fold({
                         enrollmentResultFlow.emit(Either.Left(it))
                     }, {
-                        if (it is E2EIEnrollmentResult.Initialized) requestOAuthFlow.emit(it)
-                        else enrollmentResultFlow.emit(Either.Right(it))
```
Comment (MohamadJaara) [inline app/src/main/kotlin/com/wire/android/ui/e2eiEnrollment/GetE2EICertificateViewModel.kt:72]: question: how is this change related to the fix?

Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/ui/e2eiEnrollment/GetE2EICertificateViewModel.kt:72]: Actually it is not. But `it` could be only `E2EIEnrollmentResult.Initialized` here, so this `if` is not needed and AS annoys me by yellow highlighting ;P

**Review by MohamadJaara — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3512](https://10.10.124.134/job/AR-build-pipeline/3512/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.2(9118011)(9118011)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.2(9118011)(9118011)-staging-compat.apk)
